### PR TITLE
Overview rest

### PR DESCRIPTION
**Pull request by michaelgub**
Overview Rest Call finished.

### CODE REVIEW

File: ase/src/main/java/com/ase/ase/rest/response/Overview.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.Date;
+
+
+public class Overview {
```
Comment (s1lobnig) [inline ase/src/main/java/com/ase/ase/rest/response/Overview.java:12]: This class is not needed anymore. Delete it

**Comment by Manuel57** [comment]
The annotations are still not what they should be like it is in the other controllers. The method needs

    @CrossOrigin
    @GetMapping(value = "...", produces = "application/json")
    @ResponseBody

And the class needs the annotation @CrossOrigin("*")

**Review by s1lobnig — CHANGES_REQUESTED** [review]

**Comment by michaelgub** [comment]
Changes are done.

**Comment by Manuel57** [comment]
Please do not close the pull requests all the time! The PR gets clodes by the reviewer when it is merged.

**Comment by Manuel57** [comment]
> Changes are done.

Still not correct. Check the requestPath and remove the "}"

**Comment by michaelgub** [comment]
all changes done and merged with the master.

**Comment by michaelgub** [comment]
I had to close it because I needed to merge with the master.

**Comment by michaelgub** [comment]
In my eyes there is no "}" to remove because the build passes. I just edited the format now. You have to be specific about the requestPath don't know what you mean because /overview is correct.

**Pull request merged**